x-pack/plugins/security_solution/server/lib/detection_engine/rule_schema/model/rule_schemas.ts

```diff
@@ -237,6 +238,7 @@ export const ThresholdSpecificRuleParams = z.object({
   savedId: SavedQueryId.optional(),
   threshold: ThresholdNormalized,
   dataViewId: DataViewId.optional(),
+  alertSuppression: z.object({ duration: AlertSuppressionDuration }).optional(),
```

Comment:
Doesnt it make sense to import and use `ThresholdAlertSuppression` here, directly from `threshold_attributes.gen.ts`?

(`ThresholdAlertSuppression` is also unused otherwise)

Reply:
Yes! that was leftovers prior to small refactoring, thanks for catching this